### DISCUSSION TITLE
What does the preset field (ultraslow to ultrafast) do, or should do?

### DISCUSSION THREAD

**Discussion by vnicolici** [deforum-art/sd-webui-text2video#113] [Q&A]
I assumed it would make the action faster or slower, but as far as I can tell it doesn't do anything, at least with ModelScope. The videos are way too fast, regardless of that setting. I have to watch them on 0.25 speed, so basically 3.5 FPS for a 15 FPS video to make any sense of them.

**Comment by hithereai** [comment]
Google:

Ffmpeg crf, ffmpeg preset.

And reduce the output video fps from 15 to something lower?

**Reply by vnicolici** [reply]
Thanks! I'll also try specifying "slowly" in the prompt, but I'm not sure it will help.

**Reply by pmonck** [reply]
Try "(slow-motion:1.4)" at the start of your prompt, adjusting the weight as you wish.

**Comment by kabachuha** [comment]
If you want to watch a video with lower fps, I'd recommend you to frame interpolate it (in Deforum's Output tab) with the 'Slow-mode' enabled instead, or else it will be too jittery to you
